src/Generators/FactoryGenerator.php

```diff
@@ -23,29 +21,11 @@
  *
  * @author Graham Campbell <graham@alt-three.com>
  * @author Scott Robertson <scottymeuk@gmail.com>
+ * @author Michael Bodnarchuk <davert@codeception.com>
  */
-class FactoryGenerator implements GeneratorInterface
+class FactoryGenerator extends EntityGenerator
 {
-    /**
-     * The kind of attribute that will be generated.
-     *
-     * @var string
-     */
-    private $kind;
-
-    /**
-     * The model instance.
-     *
-     * @var object
-     */
-    private $model;
-
-    /**
-     * The factory muffin instance.
-     *
-     * @var \League\FactoryMuffin\FactoryMuffin
-     */
-    private $factoryMuffin;
+    const PREFIX = 'factory|';
```

Comment:
No. In this case, we should implement a method to get the prefix and have that on the interface.